src/main/java/com/rocana/lucene/codec/v1/RocanaIntersectTermsEnum.java

```diff
@@ -35,7 +35,29 @@
 import org.apache.lucene.util.fst.FST;
 import org.apache.lucene.util.fst.Outputs;
 
-/** This is used to implement efficient {@link Terms#intersect} for
+/**
+ * Fork of Lucene's {@link org.apache.lucene.codecs.blocktree.IntersectTermsEnum}.
+ *
+ * Why we forked:
+ *   - To use other forked classes, like {@link RocanaFieldReader}.
+ *
+ * What changed in the fork?
+ *   - Use the other forked classes.
+ *   - Removed trailing whitespace.
+ *   - Changed these javadocs.
+ *
+ * This is one of the forked classes where no logic changed, but to get
+ * the fork to compile we had to fork this class too. That happened with
+ * several classes because they had a hard reference to another class we
+ * forked. Ideally, our forked classes would extend the original Lucene
+ * class and override just the methods we need to change. Unfortunately
+ * in most cases that wasn't an option because many Lucene classes are final.
+ *
+ * See ROCANA-8229
+ * @see RocanaSearchCodecV1
```

Comment:
Same comment here about referring to internal only JIRA tickets in open sourced code.